.travis.yml

```diff
@@ -1,6 +1,7 @@
+language: python
 dist: trusty
 sudo: false
-language: python
+group: deprecated-2017Q2
```

Comment:
It was a rollback test to the ancient version of travis. Not mandatory though, I reverted this.